### ISSUE TITLE
Hey v-hill, 

### ISSUE THREAD

**Issue by W-Bremer** [v-hill/gpu-price-tracker#1]
I'm a fan of the work you're doing on this price tracker. I'm learning Python and would like to make similar applications but it's tough getting started. I've been trying to use your code as somewhat of a template but theres some things I can't figure out. I was wondering if you could help me with the database aspect of the code, I've spent countless hours researching how to solve the problem I'm having and I know it must be something simple but I can't figure it out. I see that you update the code near daily so I'm hoping you'll see this message. I'd like to provide as much of my help in this program as I can as well, I'm not sure what your end goal with this is but I think it'd be cool if the eBay gpu prices were compared to gpu prices on other similar sites as well. If you're interested in helping me, my email is williambremer9@gmail.com. And just for some background info so you know I'm not just trying to steal your program, I'm 17 y/o and I just started my freshman year in university for a computer science degree.

**Comment by v-hill** [comment]
Hi there. The problem you're having, is there a specific error message you're getting that I can refer too? In terms of the end goal, there isn't really one. This project is mostly a sandbox for trying out new data engineering tools. I might at some point make a django app to display data in a webapp, or try doing some more advanced database management in MySQL/PostgreSQL. At the moment it's in a good place, as it actually runs most of the time. I idea behind the current database is to have something that isn't in memory, so that if the code crashes the data is retained. Hence having the database class which checks which data is already in the database and manages saving/loading of data.

**Comment by W-Bremer** [comment]
Yes, it seems like the database isn't being created. Here's the error message I get:

File "/Users/williambremer/Projects/webscraper/gpu-price-tracker-main/src/scraper.py", line 60, in <module>
    db = Database(conf)
  File "/Users/williambremer/Projects/webscraper/gpu-price-tracker-main/src/database.py", line 21, in __init__
    self.save_filepath = self.get_save_filepath(conf)
  File "/Users/williambremer/Projects/webscraper/gpu-price-tracker-main/src/database.py", line 30, in get_save_filepath
    datbaase_path = conf['paths']['filepath']
KeyError: 'filepath'

**Comment by v-hill** [comment]
Okay, I'd recomend doing a git pull on the latest code base, or making a new clone of the repo. Then make sure you edit the configuration.toml to fit your local file system as seen in the readme. That should get you further
